module/entityAssessors/helper.js

```diff
@@ -295,5 +309,32 @@ module.exports = class entityAssessorHelper {
         })
     }
 
+    static getInternalUserIdByExternalId(token, loginId) {
```

Comment:
Pass loginIds as an array in this function, no point having class property as this.externalIdToUserIdMap in a helper class file.

Function should take loginId as array return the mapping with keycloak user id.

@ajithtunerlabs

Reply:
addressed.